ui/src/pages/project-details/pipelines-select.tsx

```diff
@@ -1,7 +1,16 @@
 import { usePipelines } from 'data-services/hooks/pipelines/usePipelines'
 import { Select } from 'nova-ui-kit'
+import { useMemo } from 'react'
 import { useParams } from 'react-router-dom'
 
+// TODO: Define these options backend side?
+const DEFAULT_PIPELINE_IDS = [
```

Comment:
Perhaps not optimal, but when project ID is not specified (which is the case for a new projects), the backend is returning the full list of pipelines.